lrspline/__init__.py

```diff
@@ -389,16 +393,30 @@ def __mul__(self, x):
     def __rmul__(self, x):
         return self * x
 
+    def __getitem__(self, i):
+        return self.basis[i].controlpoint
+
 
 class LRSplineSurface(LRSplineObject):
 
-    def __init__(self, arg=None):
-        if isinstance(arg, raw.LRSurface):
+    def __init__(self, *args):
+        if len(args) == 0:
+            w = raw.LRSurface()
+        elif isinstance(args[0], raw.LRSurface):
             w = arg
+        elif isinstance(args[0], io.IOBase):
```

Comment:
This branch also supports strings and bytestrings, so I guess

```python
elif isinstance(args[0], (io.IOBase, str, bytes)):
```

Reply:
Was not aware of this. Will fix